packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/integ.start-job-run.ts

```diff
@@ -0,0 +1,105 @@
+//import * as iam from '@aws-cdk/aws-iam';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as eks from '@aws-cdk/aws-eks';
+import { AwsAuthMapping } from '@aws-cdk/aws-eks';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { Aws } from '@aws-cdk/core';
+import { EmrContainersStartJobRun } from '../../lib';
+import { ReleaseLabel, VirtualClusterInput } from '../../lib/emrcontainers/start-job-run';
+
+/**
+ * Stack verification steps:
+ * Everything in the link below must be setup before running the state machine.
+ * @see https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-enable-IAM.html
+ * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn> : should return execution arn
+ * aws stepfunctions describe-execution --execution-arn <exection-arn generated before> : should return status as SUCCEEDED
```

Comment:
can you just confirm here that you have run through these stack verification steps and that this works. 
i am still most concerned about the custom resource portion. do you have a way of determining that the updated role policy works successfully?

Reply:
I have verified that the test works by deploying and executing the state machine. The test attempts to run a job by using the updated role policy.